core/src/test/java/io/narayana/nta/test/HandlerServiceTest.java

```diff
@@ -79,6 +79,7 @@ public static WebArchive createDeployment() {
                 .addAsWebInfResource(new FileAsset(new File("src/test/resources/persistence.xml")),
                         "classes/META-INF/persistence.xml")
                 .addAsManifestResource(new FileAsset(new File("src/test/resources/nta-test-ds.xml")), "nta-test-ds.xml")
+                .addAsManifestResource(new FileAsset(new File("src/test/resources/filter.properties")), "filter.properties")
```

Comment:
I don't understand why it need to do here. It just create the files in the test.war used by the arquillian to deploy to the container.